homeassistant/components/vesync/entity.py

```diff
@@ -46,9 +53,12 @@ def device_info(self) -> DeviceInfo:
             sw_version=self.device.current_firm_version,
         )
 
-    def update(self) -> None:
-        """Update vesync device."""
-        self.device.update()
+    async def async_added_to_hass(self) -> None:
+        """When entity is added to hass."""
+        await super().async_added_to_hass()
+        self.async_on_remove(
+            self.coordinator.async_add_listener(self.async_write_ha_state)
+        )
```

Comment:
Why do we do this?

Reply:
I don't think we need to take any additional action for this integration on remove. I have removed it.